File: Lib/Telerik/Examples/QuickStart/Docking/MDI/CustomerForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Telerik.Examples.WinControls.Docking.MDI
{
    public partial class CustomerForm : Form
    {
        public CustomerForm()
        {
            InitializeComponent();
        }

        private void customersBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.customersBindingSource.EndEdit();
            this.customersTableAdapter.Update(this.nwindRadGridView.Customers);

        }

        protected override void OnLoad(EventArgs e)
        {
            base.OnLoad(e);

            // TODO: This line of code loads data into the 'nwindRadGridView.Customers' table. You can move, or remove it, as needed.
            this.customersTableAdapter.Fill(this.nwindRadGridView.Customers);
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/Docking/MDI/CustomerForm.vb
Imports System.ComponentModel
Imports System.Text

Namespace Telerik.Examples.WinControls.Docking.MDI
	Partial Public Class CustomerForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub customersBindingNavigatorSaveItem_Click(ByVal sender As Object, ByVal e As EventArgs)
			Me.Validate()
			Me.customersBindingSource.EndEdit()
			Me.customersTableAdapter.Update(Me.nwindRadGridView.Customers)

		End Sub

		Protected Overrides Sub OnLoad(ByVal e As EventArgs)
			MyBase.OnLoad(e)

			' TODO: This line of code loads data into the 'nwindRadGridView.Customers' table. You can move, or remove it, as needed.
			Me.customersTableAdapter.Fill(Me.nwindRadGridView.Customers)
		End Sub
	End Class
End Namespace